### PR TITLE
master <- dev

### PR DESCRIPTION
**Pull request by misternebula**

### CODE REVIEW

File: QSB/QSB.cs [hunk 1]
```diff
@@ -47,6 +48,24 @@ private void Start()
             gameObject.AddComponent<GeyserManager>();
             gameObject.AddComponent<OrbManager>();
             gameObject.AddComponent<QSBSectorManager>();
+            gameObject.AddComponent<ConversationManager>();
+
+            Helper.Events.Unity.RunWhen(() => PlayerData.IsLoaded(), RebuildSettingsSave);
+        }
+
+        private void RebuildSettingsSave()
```
Comment (amazingalek) [inline QSB/QSB.cs:56]: does this fix a bug?

File: QSB/default-config.json [hunk 2]
```diff
@@ -3,6 +3,6 @@
   "settings": {
     "defaultServerIP": "localhost",
     "port": 7777,
-    "debugMode": false
+    "debugMode": true
```
Comment (amazingalek) [inline QSB/default-config.json:6]: sure?

File: QSB/QSB.cs (hunk 1 above)
Comment (misternebula) [reply QSB/QSB.cs:56]: yeah. makes sure time isnt frozen in conversations

File: QSB/default-config.json (hunk 2 above)
Comment (misternebula) [reply QSB/default-config.json:6]: just for while i'm developing it. for the final commit i change it back

**Pull request merged**